problem_builder/tests/integration/test_progression.py

```diff
@@ -34,8 +34,7 @@ def assert_warning(self, warning_dom, link_href):
         """
         self.assertEqual(warning_dom.text, 'You need to complete the previous step before attempting this step.')
         warning_link = warning_dom.find_element_by_xpath('./*')
-        link_href = 'http://localhost:8081{}'.format(link_href)
-        self.assertEqual(warning_link.get_attribute('href'), link_href)
+        self.assertTrue(warning_link.get_attribute('href').endswith(link_href))
```

Comment:
Nit: If this fails, the new version gives a rather unspecific error message ("expected True, but found False", or something like that).  I suggest adding a message like "expected link ending in {link_href} but found {href}".